apps/pdc-frontend/src/styles/globals.css

```
@@ -6,11 +6,28 @@ body {
   padding-inline: 0;
 }
 
+h1:has(strong) {
+  --utrecht-emphasis-strong-font-weight: var(--utrecht-heading-1-font-weight);
+}
+h2:has(strong) {
+  --utrecht-emphasis-strong-font-weight: var(--utrecht-heading-2-font-weight);
+}
+h3:has(strong) {
+  --utrecht-emphasis-strong-font-weight: var(--utrecht-heading-3-font-weight);
+}
+h4:has(strong) {
+  --utrecht-emphasis-strong-font-weight: var(--utrecht-heading-4-font-weight);
+}
+h5:has(strong) {
+  --utrecht-emphasis-strong-font-weight: var(--utrecht-heading-5-font-weight);
+}
+h6:has(strong) {
+  --utrecht-emphasis-strong-font-weight: var(--utrecht-heading-6-font-weight);
+}
+
```

Wat denk je: is de huidige complexiteit nodig? Ik zie 2 stukjes code die misschien niet nodig zijn:

1. De expliciete verwijzing naar de font-weight van de parent heading hoeft niet denk ik, want `inherit` zou hetzelfde effect moeten hebben.
2. De regel alleen van toepassing maken voor headings waar een strong in zit, lijkt me een onnodige optimalisatie.

Zou het niet zo simpel kunnen zijn?


```suggestion
h1,
h2,
h3,
h4,
h5,
h6 {
  --utrecht-emphasis-strong-font-weight: inherit;
}

```